Connectors/Revit/Speckle.Connectors.RevitShared/HostApp/LinkedModelHandler.cs

```diff
@@ -38,6 +45,38 @@ public List<Element> GetLinkedModelElements(ISendFilter sendFilter, Document lin
       }
       return new List<Element>();
     }
+
+    // send mode → Views (taken from the legacy code)
+    if (sendFilter is RevitViewsFilter viewFilter && viewFilter.GetView() != null)
+    {
+#if REVIT2024_OR_GREATER
+      // revit 2024 and 2025 we can use the three-parameter constructor to get only visible elements
+      RevitLinkInstance linkInstance = FindLinkInstanceForDocument(
+        linkedDocument,
+        _revitContext.UIApplication.NotNull().ActiveUIDocument.Document
+      );
+      using var viewCollector = new FilteredElementCollector(
```

Comment:
make sure this logics run on main thread, i can not see in github where they called